### DISCUSSION TITLE
Fundamentos do ASP.NET 6

### DISCUSSION THREAD

**Discussion by GabrielHFerraz** [balta-io/forum#509] [Tira Dúvidas] [answered]
Olá, estou fazendo o curso de  Fundamentos do ASP.NET 6, estou usando o versão dotnet 7,  criei uma rota no controller para capturar os usuários, quando consulto a rota recebo o seguinte erro, é algo devido a utilização da versão 7  ? 

**Mensagem de erro:** 
An unhandled exception occurred while processing the request.
InvalidOperationException: No service for type 'Web.Data.webDbContext' has been registered.

Fiz um teste consumindo as informações do banco com o EF e funcionou corretamente 

Implementação do Context
![image](https://github.com/balta-io/forum/assets/45365105/de158e0b-6e1b-441a-a375-c101a0e5c050)

Implementação do Controller 
![image](https://github.com/balta-io/forum/assets/45365105/63fa0282-3c5c-47a7-bdd9-68c3b4877568)

**Comment by henriqueweege** [comment, marked as answer]
Oi, Gabriel! Tudo certo?
Esse erro em questão diz respeito a injeção de dependência. Traduzindo, o que ele está te falando é que há alguma parte da tua aplicação que está esperando receber uma instância da classe "webDbContext", e o ASP.NET não soube resolver essa dependência.
Não se esqueça de adicionar a classe ao container de injeção de dependência do ASP.NET. Esse artigo aqui pode te ajudar: 
https://balta.io/blog/dependency-injection

Valeu!

**Reply by andrebaltieri** [reply]
👏

**Comment by GabrielHFerraz** [comment]
Show de bola, obrigado pela orientação 👊
